### ISSUE TITLE
SuperSteam not working on storefront

### ISSUE THREAD

**Issue by Compizfox** [dbalz3/SuperSteam_Firefox#1]
Hi,

I just installed your addon. Thanks for doing this, since Enhanced Steam dropped support for Firefox.

Anyway, the addon seems to work fine on https://steamcommunity.com/ (I get the green thingie at the top right of the page that says "Super Steam ready") but it doesn't do anything on http://store.steampowered.com/. I don't get the green thingie and nothing seems to be changed on the page.

I'm using Firefox Dev Edition btw, if that makes any difference.

**Comment by dbalz3** [comment]
I downloaded the Firefox Dev Edition and tried to run both the extension from the add on store and the one I am developing locally and had no problems seeing the features on the store pages.  

One of the API calls may be timing out and preventing the store page features from loading correctly.

What version number of the Dev edition are you using?  The most up-to-date one?  
Other people are having this issue as well and I am trying to get to the bottom of it.

Also, if you clear your cache and localStorage then refresh, does the issue persist?
You can clear your localStorage by going to the firefox console and typing localStorage.clear()

Let me know.

Thanks

**Comment by Compizfox** [comment]
> What version number of the Dev edition are you using? The most up-to-date one?

Yep. I'm currently using 49.0a2 but I believe I had the problem with 48 as well.

>  Other people are having this issue as well and I am trying to get to the bottom of it.

At least I'm not the only one then ;) Do those other people also use Firefox Dev Edition?

> Also, if you clear your cache and localStorage then refresh, does the issue persist?
> You can clear your localStorage by going to the firefox console and typing localStorage.clear()

No, unfortunately the problem persists.

I must also note that I use e10s, but opening the Steam Store in a non-e10s window doesn't change anything, so I doubt that is the problem.

**Comment by Compizfox** [comment]
I don't know since when exactly, but SuperSteam is now working for me.

**Comment by scooterbaga** [comment]
The addon hasn't been working for me for quiet a while now. Even after I clear the cache, the storefront pages all show the little red error dot.

The tooltip says "error loading supersteam data"

Wasn't sure if I should start a new issue as it appears to be very similar. Thanks.
